### ISSUE TITLE
Can't load webpage

### ISSUE THREAD

**Issue by porkyneal** [dbuezas/arduino-web-oscilloscope#2]
This looks very interesting, however, I have done as it says and enabled the chrome://flags/#enable-experimental-web-platform-features flag, restarted page / browser and still unable to get the page to load past telling me to enable the flag.

Can't see any console logging that I can provide, any other suggestions?

Chrome Version 88.0.4324.182 on https://dbuezas.github.io/arduino-web-oscilloscope/

![image](https://user-images.githubusercontent.com/8297625/108707355-c8dbc900-7507-11eb-9d81-0c92fe7b136c.png)

![image](https://user-images.githubusercontent.com/8297625/108707376-d002d700-7507-11eb-864c-2485181482e7.png)

**Comment by w-A-L-L-e** [comment]
I've got Version 88.0.4324.182 on macos big sur and it works without issues. However firefox does not work, will this be supported in the future?

**Comment by seisfeld** [comment]
> However firefox does not work, will this be supported in the future?

To my knowledge Firefox does not offer the required API. Currently this is only possible with Chrome, Chromium and AFAIK Microsoft Edge.

**Comment by w-A-L-L-e** [comment]
Thx for the quick reply, indeed I googled it and that feature for webserial in firefox was discussed somewhere in 2014 already but it got parked and nobody seems to continue work on it. Unfortunate, because it indeed opens up a lot of cool things to support in the browser like this one. But so many other things could be controlled with serial, like 3d printers etc.Honestly, I didn't know it even existed for chrome and edge until I saw your project on hackaday ;).
